nx.json

```diff
@@ -10,6 +10,16 @@
   "targetDependencies": {
     "release:phase2": [{"target": "release:phase2", "projects": "dependencies"}]
   },
+  "targetDefaults": {
+    "build": {
+      "dependsOn": ["^build"],
+      "outputs": [
+        "{projectRoot}/lib",
+        "{projectRoot}/dist",
+        "{projectRoot}/tsconfig.build.tsbuildinfo"
+      ]
```

Comment:
this is used for the caching of nx. It helps `nx` check/understand if a project is already compiled.